test/silhouette.jl

```diff
@@ -34,4 +34,18 @@ c = [3, 1]
 
 @test silhouettes(a, c, D) ≈ [0.5, 0.5, -1/3, 0.0]
 
+a = [fill(1, 5); fill(2, 5)]
+d = fill(1, (10, 10))
+[d[i, i] = 0 for i=1:10]    
+
+@test silhouettes(a, d) ≈ fill(0.0, 10)
+
+d[1, 2] = d[2, 1] = 50
```

Comment:
What is it testing now exactly? Do we need it?
Also, you can use `5` instead of `50` to get `-0.5` for the silhouette and do an exact comparison.